src/NServiceBus.RavenDB/Subscriptions/SubscriptionPersister.cs

```diff
@@ -38,7 +39,7 @@ public SubscriptionPersister(IDocumentStore store)
                 {
                     using (var session = OpenAsyncSession())
                     {
-                        var subscriptionDocId = Subscription.FormatId(messageType);
+                        var subscriptionDocId = Subscription.FormatId(messageType, DisableSubscriptionsVersioning);
```

Comment:
Seeing these repeated instances of passing the boolean to FormatId makes me think we could surround FormatId with an object that determines its format at startup time so there's not so many opportunities to get the branching incorrect. I fully acknowledge that might be YAGNI, haven't thought it through enough.